src/utils/setup/remove-excluded-articles.js

```diff
@@ -0,0 +1,25 @@
+const removeExcludedArticles = (allArticles, excludedLearnPageArticles) => {
+    if (excludedLearnPageArticles && excludedLearnPageArticles.length) {
+        const filteredArticles = allArticles.filter(
+            article =>
+                !excludedLearnPageArticles.find(excludedArticle =>
+                    article.query_fields.slug.match(
+                        new RegExp(`^/?${excludedArticle}$`)
+                    )
+                )
+        );
+        // Warn writers if not all excludes were found
+        if (
```

Comment:
This seemed like a nice touch, but not sure if it is needed (or something we want to remove after this works for them).